ui/app/com/gu/recipeasy/controllers/Application.scala

```diff
@@ -29,7 +29,7 @@ class Application(override val wsClient: WSClient, override val conf: Configurat
         val curatedRecipe = CuratedRecipe.fromRecipe(r)
         val curatedRecipeForm = CuratedRecipeForm.toForm(curatedRecipe)
         db.setRecipeStatus(r.id, "Pending")
-        Ok(views.html.recipe(createCuratedRecipeForm.fill(curatedRecipeForm), r.id, r.body, r.articleId))
+        Ok(views.html.recipe(createCuratedRecipeForm.fill(curatedRecipeForm), r.id, r.body, r.articleId, true))
```

Comment:
For readability this would be better written as: 
Ok(views.html.recipe(createCuratedRecipeForm.fill(curatedRecipeForm), r.id, r.body, r.articleId, shouldShowButtons = true))